scripts/actions/add-files-to-translation-queue.js

```diff
@@ -98,6 +102,7 @@ const main = async () => {
   const url = options.url || null;
   const directory = options.directory || null;
   const machineTranslation = options.machineTranslation || false;
+  const localeOption = options.locale || null;
```

Comment:
looking at the usage below, do we need this separate variable here?

can we have:
```
if (options.locale)
```
on line 138